lib/ansible/utils/module_docs_fragments/shell_common.py

```diff
@@ -11,7 +11,7 @@ class ModuleDocFragment(object):
     description:
       - Temporary directory to use on targets when executing tasks.
     default: '~/.ansible/tmp'
-    env: [{name: ANSIBLE_REMOTE_TEMP}]
+    env: [{name: ANSIBLE_REMOTE_TMP}, {name: ANSIBLE_REMOTE_TEMP}]
```

Comment:
+10

Reply:
Oh, I thought it was first found but the code is last found.  Will change.